Assets/Scripts/InventoryAnimation.cs

```diff
@@ -3,8 +3,10 @@
 public class InventoryAnimation : MonoBehaviour
 {
     private RectTransform _rectTransform;
-    private bool isShowing = false;
-
+    private bool _isShowing = false;
+    public bool isMagnifierSelected { get; set; }
```

Comment:
the properties start with capital letters
```suggestion
    public bool IsMagnifierSelected { get; set; }
```